### PR TITLE
Dummy ui

### PR DESCRIPTION
**Pull request by andreibancioiu**
Serve web files (web folder), sketch UI, bind it to our IOU controller.

An internal error is returned at first call to GetNotesSentByMe() - just ignore it (continue execution).
That bug will be fixed in another commit.

### CODE REVIEW

File: src/IOUService/App.config [hunk 1]
```diff
@@ -1,6 +1,14 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (ctrucza) [inline src/IOUService/App.config:1]: Ugh.

File: src/IOUService/packages.config [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 <packages>
-  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.2" targetFramework="net45" />
-  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.2" targetFramework="net45" />
-  <package id="Microsoft.AspNet.WebApi.Owin" version="5.2.2" targetFramework="net45" />
-  <package id="Microsoft.AspNet.WebApi.OwinSelfHost" version="5.2.2" targetFramework="net45" />
-  <package id="Microsoft.Owin" version="2.0.2" targetFramework="net45" />
-  <package id="Microsoft.Owin.Host.HttpListener" version="2.0.2" targetFramework="net45" />
-  <package id="Microsoft.Owin.Hosting" version="2.0.2" targetFramework="net45" />
-  <package id="Newtonsoft.Json" version="6.0.4" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net45" />
```
Comment (ctrucza) [inline src/IOUService/packages.config:3]: Updating libraries should be done with more consideration.

**Comment by ctrucza** [comment]
Two issues (see comments above):
- space changes should be taken care separately.
- changing the versions of the used libraries is a Big Thing (TM). It should not be done casually.

File: src/IOUService/App.config (hunk 1 above)
Comment (andreibancioiu) [inline src/IOUService/App.config:1]: Sorry for that...

File: src/IOUService/packages.config (hunk 2 above)
Comment (andreibancioiu) [inline src/IOUService/packages.config:3]: For this, too :)

**Pull request merged**